_posts/2018-10-25-released-v5.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Released Chainer/CuPy v5.0.0
+layout: post
+categories: Announcement
+author: Seiya Tokui
+---
+
+We have released Chainer and CuPy v5.0.0 today!
+This is a major release that introduces several new features.
+
+The following is a list of selected updates. Full updates can be found in the release notes: [Chainer](https://github.com/chainer/chainer/releases/tags/v5.0.0), [CuPy](https://github.com/cupy/cupy/releases/tag/v5.0.0).
+
+- **Static subgraph optimization** *(experimental)*.
+  By applying `@static_graph` decorator to the static part of your computation (which uses the same graph at every iteration), the computational graph of that part is cached and reused.
+  For fully-static models, we are observing 20-60% speed up in most cases.
+  Example code modified for the static subgraph feature can be found [here](https://github.com/chainer/chainer/tree/v5/examples/static_graph_optimizations).
+- **Float16 support**.
+  Using half-precision float is made much easier!
+  Since recent GPU technologies are focusing on half and mixed precision computations, using float16 is crucial for fully utilizing the latest hardware performance.
```

Comment:
```suggestion
  Since recent GPU technologies often focu on half and mixed precision computations, using float16 is crucial for fully utilizing the latest hardware performance.
```